os/fs/smartfs/smartfs_smart.c

```diff
@@ -1259,7 +1259,7 @@ static int smartfs_bind(FAR struct inode *blkdriver, const void *data, void **ha
 	}
 
 	/* Initialize the allocated mountpt state structure.  The filesystem is
-	 * responsible for one reference ont the blkdriver inode and does not
+	 * responsible for one reference on the blkdriver inode and does not
```

Comment:
You can add below change as well at line:1249
/* If the global semaphore hasn't been initialized, then
	 * initialized it now. */  => initialize